source/lib/JMSLab/tests/test_build_latex.py

```diff
@@ -13,17 +13,19 @@
 
 from ..builders.build_latex import build_latex, LatexBuilder
 from .._exception_classes import ExecCallError
+from .. import misc
```

Comment:
@ew487 Per spyder, this package is imported but not used. Might want to double check.

Reply:
@ew487 Done in https://github.com/JMSLab/Template/pull/89/commits/11a1c76e2a5668238b3bdb2fd1abd0e10d892a49. Tests ran as usual after the removal so I am resolving.

Reply:
@rcalvo12 Great, thank you!